SlickSolutions/slicksolutions/convex/stripe.ts

```
@@ -1,132 +1,270 @@
-          stripeCurrentPeriodEnd: subscription.current_period_end * 1000,
-        });
+  const session = event.data.object as Stripe.Checkout.Session;
+  const subscription = event.data.object as Stripe.Subscription;
+
+  // Handle the event
+  switch (event.type) {
+    case 'checkout.session.completed':
+      console.log('checkout.session.completed', session);
+      const completedSession = await stripe.checkout.sessions.retrieve(session.id, {
+        expand: ['line_items'],
+      });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)